src/taipy/templates/scenario-management/cookiecutter.json

```
@@ -0,0 +1,11 @@
+{
+    "Your application root folder name": "taipy_application",
+    "Your application main Python file": "main.py",
+    "Your application title": "Taipy Application",
+    "Use TOML Config": "Y/N",
```

Usually, the Capital letter is the default value. That s why I had put y/N.
We can put the expected value directly [No], so the user can understand he needs to put Yes.

```suggestion
    "Use TOML Config": "No",
```
 I am not sure about my proposal, to be honest.